src/Repository/ElectedRepresentative/ElectedRepresentativeRepository.php

```diff
@@ -82,11 +83,31 @@ private function withZoneCondition(QueryBuilder $qb, array $referentTags, string
             $qb->leftJoin($alias.'.mandates', 'mandate');
         }
 
-        return $qb
+        $hasParis = false;
+        foreach ($referentTags as $tag) {
+            if (0 === mb_strpos($tag->getName(), 'Paris')) {
+                $hasParis = true;
+
+                break;
+            }
+        }
+
+        $zoneCondition = new Orx();
+        $zoneCondition->add('tag IN (:tags)');
+        $qb->setParameter('tags', $referentTags);
+        // if referent has some Paris tag, we should return elected representatives of all Paris zones
+        if ($hasParis) {
+            $zoneCondition->add('tag.code LIKE :paris_arr OR tag.code LIKE :paris_circo');
+            $qb->setParameter('paris_arr', '750%');
+            $qb->setParameter('paris_circo', 'CIRCO_750%');
```

Comment:
this can lead to unexpected behavior I think because `_` in a `LIKE` means `one char no matter which`. I think we need to escape it here